flink-core/src/main/java/org/apache/flink/core/fs/AbstractCloseableRegistry.java

```diff
@@ -16,29 +16,29 @@
  * limitations under the License.
  */
 
-package org.apache.flink.runtime.state;
+package org.apache.flink.core.fs;
```

Comment:
Since this is used for other things than filesystems, how about moving it to `org.apache.flink.core.util`?